client/src/router/index.js

```diff
@@ -0,0 +1,23 @@
+import { createRouter, createWebHistory } from "vue-router";
+import HomeView from "../views/HomeView.vue";
```

Comment:
Same alias question for this and the import on line 18.

Reply:
Alias can be used here too.